store/policy.go

```diff
@@ -291,12 +285,6 @@ func (s *Store) composePolicy(ctx context.Context, raw *policyRaw) (*api.Policy,
 // getPolicyRaw finds the policy for an environment.
 // Returns ECONFLICT if finding more than 1 matching records.
 func (s *Store) getPolicyRaw(ctx context.Context, find *api.PolicyFind) (*policyRaw, error) {
-	// Validate policy type existence.
```

Comment:
Seems we do not allow users to query the policy without the policy type, how about using `PolicyType` instead of the pointer for the `Type` field in the `PolicyFind`?

Reply:
True, done.